tutoropenstack/plugin.py

```diff
@@ -27,12 +27,7 @@
     },
 }
 
-hooks.filters.add_items(
-    "cli:commands",
-    [
-        openstack_command,
-    ],
-)
+hooks.Filters.CLI_COMMANDS.add_item(openstack_command)
```

Comment:
So just checking: do we know that this is a non-breaking, backwards-compatible change that doesn't at all change the minimum Tutor version required? In other words, will this really work with _any_ Nutmeg Tutor version all the way back to 14.0.0?